src/tracing/errors.ts

```diff
@@ -9,45 +9,51 @@ declare global {
   }
 }
 
-/** Identifies ethers request errors (as thrown by {@type import(@ethersproject/web).fetchJson}). */
+// Identifies ethers request errors (as thrown by {@type import(@ethersproject/web).fetchJson}).
```

Comment:
nit: This is a doc-comment, not an implementation comment, even though it's not exported, so you should prefer JSDoc syntax: `/**` instead of `//`.

Reply:
idk if there's a way to lint for this also?